### ISSUE TITLE
pyTorch 1.4 does not support torch.square

### ISSUE THREAD

**Issue by rhewett** [distdl/distdl#161]
The new batchnorm tests fail on torch 1.4 because torch 1.4 does not have torch.square.

We might shift to requiring torch 1.6 shortly, but for now this needs to be fixed.
